go-controller/pkg/ovn/admin_network_policy_test.go

```diff
@@ -2066,4 +1976,124 @@ var _ = ginkgo.Describe("OVN ANP Operations", func() {
 			gomega.Expect(err).ToNot(gomega.HaveOccurred())
 		})
 	})
+	ginkgo.Context("Multiple ANPs at the same priority", func() {
```

Comment:
Would it be a good idea to add a test case where 1) one of the 2 duplicate priorities are "not loaded" and then 2) the currently loaded priority is "unloaded / deleted" and 3) ensure that the unused admin netw policy is not used and enforced?
I also don't know if you had already tested cases when the duplication issue is resolved and the policy that was not chosen beginc taking effect. Just a question to ensure that is taken care of!

Reply:
oh so this was the old behaviour @flavio-fernandes where we were having two ANPs where we had a passive active mode, now what we plan to do is to allow to apply both the ANPs at the same priority. So moving forward both the policies will be chosen actually.

Reply:
yes, it does. Thanks!